### ISSUE TITLE
switch to opentelemetry from opentracing once it is ready

### ISSUE THREAD

**Issue by stevehu** [networknt/light-4j#565]
The OpenTracing and OpenCensus have been merged into a new project OpenTelemetry. It is still in heavy development and we will switch to it once it is ready.

**Comment by stevehu** [comment]
Close it for now until we can take action.

**Comment by breedx-splk** [comment]
It's probably worth a look. Tracing is quite stable these days. 🥂
